### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by kylewilliams321** [Aohzan/hass-blueprints#12]
I use this blueprint and for the most part really like it as a daily reminder to take my meds.

however, I think there is one improvement that will make this much better in my eyes.

Situation: Sometimes I wake up early and right before brushing my teeth I'll just take my meds then... and then I go into HA and click my input boolean that I took the meds already.  I expected this to then turn off the first run of the daily reminders as there is no need to send reminder because I already took the meds.

However, this doesn't appear to be the case.  I have 9am as the set start time and right at 9am the input boolean resets to off and the automation runs reminding me to take the meds.

So I propose a way to turn it off ahead of time if you wanted to take your meds already and didn't need to be reminded.

Thanks!

**Comment by Aohzan** [comment]
currently, it turn off the input_boolean at the beginning, if I remove this, and add a condition to not run the automation if the input_boolean is already on, everyone one should ask a new automation to reset the state of the input_boolean before the run of this blueprint

**Comment by kylewilliams321** [comment]
Right, make another input saying at what time the input Boolean should be
reset. I actually am resetting mine along with a few other non related
input booleans at 2am everyday.

But you could add in a reset time where the state goes back to off daily.

Thoughts?

**Comment by Aohzan** [comment]
I don't think it will work because it is another automation

**Comment by kylewilliams321** [comment]
Ohhh so it can’t do two things?
1) everything as before
2) also turn off automation at a specific time?
